data/libs/autoload.lua

```diff
@@ -8,6 +8,16 @@ string.trim = function(s)
 	return string.gsub(s or "", "^%s*(.-)%s*$", "%1")
 end
 
+string.scase = function (s)
```

Comment:
This function needs a short documentation string indicating its function, since the name is no longer descriptive.

```suggestion
-- Convert the input string to s(entence) case, capitalizing the
-- first letter of the string and any first letters of consecutive
-- sentences in the string.
string.scase = function (s)
```

(EDIT: manual word-wrapping in the Github interface is very difficult to determine...)